static/js/containers/ProfileFormContainer.js

```diff
@@ -177,6 +113,24 @@ class ProfileFormContainer extends React.Component {
     }
   }
 
+  simpleActionHelpers: Function = (): ActionHelpers => {
+    const { dispatch } = this.props;
+    return createSimpleActionHelpers(dispatch, [
+      [setWorkDialogVisibility, "setWorkDialogVisibility"],
+      [setWorkDialogIndex, "setWorkDialogIndex"],
+      [clearProfileEdit, "clearProfileEdit"],
+      [setEducationDialogVisibility, "setEducationDialogVisibility"],
+      [setEducationDialogIndex, "setEducationDialogIndex"],
+      [setEducationDegreeLevel, "setEducationDegreeLevel"],
+      [setUserPageDialogVisibility, "setUserPageDialogVisibility"],
+      [setShowEducationDeleteDialog, "setShowEducationDeleteDialog"],
+      [setShowWorkDeleteDialog, "setShowWorkDeleteDialog"],
+      [setDeletionIndex, "setDeletionIndex"],
+      [setShowWorkDeleteAllDialog, "setShowWorkDeleteAllDialog"],
+      [setShowEducationDeleteAllDialog, "setShowEducationDeleteAllDialog"],
```

Comment:
if that doesn't work for whatever reason, i think you can also use `func.prototype.constructor.name` or something gross like that